addons/website/static/src/js/menu/edit.js

```diff
@@ -296,10 +296,7 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
                     continue;
                 }
                 $savable.not('.o_dirty').each(function () {
-                    const $el = $(this);
```

Comment:
> > `if (!$el.closest('[data-oe-readonly]').length) {` has been removed because its existance was unclear. It does not seems right [...] If the case covered by that line reapears, it would probably need to be handled differently.
> 
> What do you mean by "its existance was unclear"?

I've corrected with "its reason to exists is unknown". I hope is clear.

> What do you mean by "reappears"?

That note was not necessary anyway so I've removed it.

Reply:
> I've corrected with "its reason to exists is unknown". I hope is clear.

I was not talking about the sentence itself but actually about your diff... Reading the code it seems to make sense: something which is readonly should not become dirty. So I am guessing you digged deeper to be sure of what you are removing. Did you check / blame when that was introduced and if the explication is not clear enough discussed with the author? Did you try to modify such a section and check that it's still well readonly? That's the kind of info I hope to find in a commit message that removes useless lines (if they are). Just "it's unclear what it is so I removed them" really is not enough (hopefully not what you did but I cannot know it from your message).

Reply:
Good that you've pushed me to digg deeper.
I've analyzed that `data-oe-readonly` can be set from python for readonly fields and in `translate.js`
```js
        $(savableSelector)
            .filter(':has(' + savableSelector + ')')
            .attr('data-oe-readonly', true);
```
This seems to be there in order to prevent saving savable elements that contains savable element (as it is not supported in python).
But it should be possible to save the savable elements that do not contain savable element.